### PR TITLE
Add support for Scala 2.13 and prepare for version to 1.0.0

### PR DESCRIPTION
**Pull request by markvandertol**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -3,7 +3,7 @@ val commonSettings = Seq(
   licenses += ("Apache License, Version 2.0", url("http://www.apache.org/licenses/LICENSE-2.0"))
 )
 
-val specs2 = "org.specs2" %% "specs2-core" % "4.4.1" % "test"
+val specs2 = "org.specs2" %% "specs2-core" % "4.8.3" % "test"
```
Comment (poohsen) [inline build.sbt:6]: why not 4.10.5?

File: build.sbt [hunk 2]
```diff
@@ -23,7 +23,8 @@ val `interpolators` = project
     name := "mobilityid-interpolators",
     commonSettings,
     libraryDependencies ++= Seq(
-      "com.propensive" %% "contextual" % "1.1.0",
+      "com.propensive" %% "contextual-core" % "3.0.0",
+      "org.scala-lang" % "scala-reflect" % scalaVersion.value,
```
Comment (poohsen) [inline build.sbt:27]: do we need this? it seems to work for me without it

File: core/src/main/scala/com/thenewmotion/mobilityid/checkDigit.scala [hunk 3]
```diff
@@ -50,7 +50,7 @@ private[mobilityid] object LookupTables {
     'R' -> 3, 'S' -> 19, 'T' -> 35,
     'U' -> 7, 'V' -> 23, 'W' -> 39,
     'X' -> 11, 'Y' -> 27, 'Z' -> 43
-  ).mapValues(decode)
+  ).map(r => (r._1, decode(r._2)))
```
Comment (poohsen) [inline core/src/main/scala/com/thenewmotion/mobilityid/checkDigit.scala:53]: hmm. migration guide said to simply add `.toMap`

File: project/build.properties [hunk 4]
```diff
@@ -1 +1 @@
-sbt.version=1.2.8
+sbt.version=1.3.10
```
Comment (poohsen) [inline project/build.properties:1]: 1.4.4 is out

File: project/plugins.sbt [hunk 5]
```diff
@@ -1,6 +1,6 @@
 resolvers += "TNM" at "https://nexus.thenewmotion.com/content/groups/public"
 
-addSbtPlugin("com.newmotion" % "sbt-build-seed" % "4.1.4")
+addSbtPlugin("com.newmotion" % "sbt-build-seed" % "5.1.0")
 
 // https://github.com/rtimush/sbt-updates
-addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.4.0")
+addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.5.1")
```
Comment (poohsen) [inline project/plugins.sbt:6]: let's take it out. we should all have it in our global sbt plugins list

File: core/src/main/scala/com/thenewmotion/mobilityid/checkDigit.scala (hunk 3 above)
Comment (markvandertol) [reply core/src/main/scala/com/thenewmotion/mobilityid/checkDigit.scala:53]: Adding .toMap works. It prints a deprecation warning, but we can ignore that until we phase out 2.12 support

Comment (poohsen) [reply core/src/main/scala/com/thenewmotion/mobilityid/checkDigit.scala:53]: then let's leave it the way you did it

Comment (markvandertol) [reply core/src/main/scala/com/thenewmotion/mobilityid/checkDigit.scala]: I switched it .mapValues().toMap anyway. It describes the intent better

File: build.sbt (hunk 2 above)
Comment (markvandertol) [reply build.sbt]: It was needed for 2.0.0, but 3.0.0 no longer. I removed it

File: build.sbt (hunk 1 above)
Comment (markvandertol) [reply build.sbt]: Updated

**Review by poohsen — APPROVED** [review]

**Pull request merged**